plugins/chatgpt.py

```diff
@@ -1,24 +1,33 @@
-from cloudbot import hook
-from cloudbot.bot import bot
 from datetime import datetime, timedelta
-import requests
 import textwrap
 import re
 import shutil
 import os
 import random
 import string
+import requests
+
+from cloudbot import hook
+from cloudbot.bot import bot
+
 
 
 RATELIMIT = {}
-SYSTEM = ""
-CONTEXT = {}
+CONTEXT = {
+    "messages": [
+        {
+        "role": "system",
+        "content": "Answer every query with a smiley face at the end.",
```

Comment:
```suggestion
        "content": "",
```

With our luck, the bot will use emoji after 👏  every 👏  word 👏

Reply:
lmao just a placeholder, I removed it.